ushahidi/platform-client-mzima
# Fix console errors

@tuxpiper @Angamanga This resolves issue [#4718](https://github.com/ushahidi/platform/issues/4718) and [#4719](https://github.com/ushahidi/platform/issues/4719)

**How To Test**

- Go to data view, filter post by archive only (inspect the console to see there are no console errors)
- Go to Settings/categories, add new, choose a parent category (inspect the console to see there are no console errors)


https://github.com/ushahidi/platform-client-mzima/assets/46582086/e641916f-784c-4d62-ab62-c3cf77c8770b
